### PR TITLE
Happy New Year! :santa: :gift: :christmas_tree:

### PR DESCRIPTION
**Pull request by sh7dm**
Merry Christmas! Happy New Year!
:santa: :gift: :christmas_tree:

### CODE REVIEW

File: deno.d.ts [hunk 1]
```diff
@@ -0,0 +1,2254 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+
+/// <reference no-default-lib="true" />
+/// <reference lib="esnext" />
+
+declare module "deno" {
```
Comment (ry) [inline deno.d.ts]: Remove this file

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "compilerOptions": { "types": ["./deno.d.ts"] }
```
Comment (ry) [inline tsconfig.json]: remove this file

File: deno.d.ts (hunk 1 above)
Comment (sh7dm) [reply deno.d.ts]: @ry Oh, I'm sorry. I'm using this for types in VSCode

File: tsconfig.json (hunk 2 above)
Comment (sh7dm) [reply tsconfig.json]: @ry Oh, I'm sorry. I'm using this for types in VSCode

**Pull request merged**